src/modules/EditProfile/index.tsx

```diff
@@ -15,17 +15,26 @@ import {
   InputsWrapper,
   ButtonWrapper,
   FormWrapper,
-  UserCourseListItem,
   UserCoursesListTitle,
 } from './styled';
 import { BG_COLOR, MAIN1_COLOR } from 'appConstants/colors';
 import { CURRENT_YEAR, SET_USER_DATA } from 'appConstants';
+import {
+  IOldCourses,
+  UserCourseListItem,
+} from './components/UserCourseListItem';
 
 export const EditProfile: FC = () => {
   const history = useHistory();
   const loginToken = useSelector(selectToken);
   const userData = useSelector(selectUserData);
-  const [userCourses, setUserCourses] = useState<Course[]>(userData.courses);
+
+  const oldCourses: IOldCourses[] = [...userData.courses].map((course) => {
```

Comment:
const oldCourses: IOldCourses[] = userData.courses.map((course: Course) => ({ ...course, isNew: true }));
Check is it works, if yes - implement.

Reply:
I tried to do that. 
![image](https://user-images.githubusercontent.com/20399054/108593254-ca7c8400-7383-11eb-942d-3a23e40f0a14.png)